corehq/apps/data_analytics/analytics.py

```diff
@@ -0,0 +1,16 @@
+from corehq.apps.es.aggregations import AggregationTerm, NestedAggregationHelper
```

Comment:
are we going with `analytics.py` as the convention or `esaccessors.py`? i see both in the repo. some of the analytics files have couch calls though